shared/scripts/rh_perf_log_hostinfo_script.sh

```diff
@@ -26,22 +26,21 @@ ports=`brctl show|grep -v "bridge.*name.*bridge.*id"|awk {'print $4'}`
 for i in $bridges;do
     call echo "ethtool -k $i"
     call ethtool -k $i
+    call brctl showstp $i
```

Comment:
do you need to redirect stderr to /dev/null, do it  no error print if $i is not a bridge.